emission/tests/analysisTests/modeinferTests/TestPipelineNew.py

```diff
@@ -0,0 +1,342 @@
+#Standard imports
+import unittest
+import json
+import logging
+import numpy as np
+from datetime import datetime, timedelta
+
+# Our imports
+from emission.core.get_database import get_db, get_mode_db, get_section_db
+import emission.analysis.classification.inference.mode as pipeline
+from emission.core.wrapper.user import User
+from emission.core.wrapper.client import Client
+import emission.tests.common as etc
+'''
+TODO:
+    Create some outliers and make sure they're stripped out
+
+'''
+class TestPipeline(unittest.TestCase):
+  def setUp(self):
+    self.testUsers = ["test@example.com", "best@example.com", "fest@example.com",
+                      "rest@example.com", "nest@example.com"]
+    self.serverName = 'localhost'
+
+    # Sometimes, we may have entries left behind in the database if one of the tests failed
+    # or threw an exception, so let us start by cleaning up all entries
+    etc.dropAllCollections(get_db())
+
+    self.ModesColl = get_mode_db()
+    self.assertEquals(self.ModesColl.find().count(), 0)
+
+    self.SectionsColl = get_section_db()
+    self.assertEquals(self.SectionsColl.find().count(), 0)
+
+    etc.loadTable(self.serverName, "Stage_Modes", "emission/tests/data/modes.json")
+    etc.loadTable(self.serverName, "Stage_Sections", "emission/tests/data/newTestModeInferFilePlacesDeleted")
```

Comment:
in the new model, data is not stored in `Stage_Sections` but instead, in the timeseries